modules/giapi/src/main/scala/giapi/client/GiapiStatusDb.scala

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2016-2018 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package giapi.client
+
+import cats.Applicative
+import cats.implicits._
+import cats.effect.Sync
+import cats.effect.Resource
+import cats.effect.Effect
+import cats.effect.ConcurrentEffect
+import cats.effect.implicits._
+import edu.gemini.aspen.giapi.status.StatusHandler
+import edu.gemini.aspen.giapi.status.StatusItem
+import edu.gemini.aspen.giapi.statusservice.StatusHandlerAggregate
+import edu.gemini.jms.activemq.provider.ActiveMQJmsProvider
+import fs2.concurrent.Queue
+import fs2.Stream
+
+/////////////////////////////////////////////////////////////////
+// Links status streaming with the giapi db
+/////////////////////////////////////////////////////////////////
+trait GiapiStatusDb[F[_]] {
+  def value(i: String): F[Option[StatusValue]]
+  def close: F[Unit]
```

Comment:
Do we want to rely on the user to call `close`? If `newStatusDb` below yielded a `Resource` it could clean itself up.

Reply:
hmm, so make it do the `close` call only in the `resource`. Makes sense